templates/toolbar.build.hbs

```diff
@@ -1 +1 @@
-<p><span class="unread-count">\{{TN count}}</span> unread <span class="toolbar-read-all">(<a data-read-all href="#">Mark all as read</a>)</span><i class="fa fa-lg fa-fw fa-cog pull-right" data-settings></i><i class="fa fa-lg fa-fw fa-refresh pull-right" data-refresh></i></p>
+<div class="switch-toggle-holder"><label class="switch"><input type="checkbox" id="notificationToggle" checked><span class="slider round"></span></label><span class="unread-label">Only show unread</span><span class="show-all-label">Show all notification</span></div><i class="fa fa-lg fa-fw fa-cog pull-right" data-settings></i><i class="fa fa-lg fa-fw fa-refresh pull-right" data-refresh></i><div class="unread"><span class="unread-count">\{{TN count}}</span> unread <span class="toolbar-read-all">(<a data-read-all href="#">Mark all as read</a>)</div></span>
```

Comment:
- `id` should never be used in widget build HTML. This is because there could be multiple instances of a widget on any page, and we can't have multiple DOM elements with the same `id`.
- `id` must use kebab case

See https://docs.google.com/document/d/1XPlQHPbqp3rGGTekcC05bKtwKDDjrU2RMJ4VIHLXp64/edit#heading=h.yewnp98kurqg